mne/report.py

```diff
@@ -652,14 +674,22 @@ class Report(object):
         Subject name.
     title : str
         Title of the report.
+    cov_fname : str
+        Name of the file containing the noise covariance.
+    baseline : str | bool
+        If True, apply baseline. If 'auto', baseline correct only if highpass
+        filter has not been applied or is less than 1.0 Hz.
```

Comment:
I'd like people to save baseline corrected data. So we avoid this magic. To
me it's a mistake that MNE delays baselining at inverse modeling /
mne_analyze stage.

I'd go for --bmin and --bmax options to set baseline and avoid magic.